tutorials/Numerics/TimeStepping/imex_ark.jl

```diff
@@ -0,0 +1,18 @@
+# # [Implicit-Explicit (IMEX) Additive Runge-Kutta Timestepping](@id Single-rate-IMEXARK-Timestepping)
+
+# Lore ipsum
+
+include(joinpath(
+    @__DIR__,
+    "../../../../../",
+    "tutorials/Numerics/TimeStepping/tutorial_acousticwave_config.jl",
+))
```

Comment:
```suggestion
using ClimateMachine
clima_dir = dirname(dirname(pathof(ClimateMachine)));
include(joinpath(clima_dir, "tutorials/Numerics/TimeStepping/tutorial_acousticwave_config.jl"))
```
I've found this to be a useful pattern :slightly_smiling_face:

Reply:
Oh, does `clima_dir` in this case points to the generated path for `climatemachine-docs`?

Reply:
`clima_dir` is a string that points to `ClimateMachine.jl/`

Reply:
Maybe we should change the name everywhere to `climate_machine_dir` (this was originally written back when ClimateMachine.jl was CLIMA.jl). Maybe for a separate PR, though.

Reply:
Thanks Charlie! I adopted your suggestion. Looks much neater.